### ISSUE TITLE
Rename the options to makes them easier to understand

### ISSUE THREAD

**Issue by AlexandreBonneau** [BobKnothe/autoNumeric#296]
This is the first proposal :
```
aSep = thousandSeparator or digitGroupSeparator
nSep = noSeparatorOnFocus 
dGroup = thousandSeparatorDigitGrouping
aDec = decimalCharacter
altDec = decimalCharacterKeyAssignment
aSign = currencySymbol
pSign = currencySymbolPlacement
pNeg = negativeSignPlacement
aSuffix = suffixText
vMax = maximumValue
vMin = minimumValue
oLimits = overrideMinMaxLimits
mDec = decimalPlacesOverride
eDec = decimalPlacesShownOnFocus
scaleDivisor 
scaleDecimal = scaleDecimalPlaces
scaleSymbol
aStor = saveValueToSessionStorage
mRound = roundingMethod
aPad = allowPadding
nBracket = negativeBracketsTypeOnBlur
wEmpty = emptyInputBehavior
lZero = leadingZero
aForm = formatOnPageLoad
sNumber = selectNumberOnly
anDefault = defaultValueOverride
unSetOnSubmit = unformatOnSubmit
outputType = outputFormat 
debug = debugMode or showWarnings
```

Any thoughts on that?

**Comment by AlexandreBonneau** [comment]
Changes proposed by Bob :
```
// digital grouping is recognized in spreadsheets and wikipedia
aSep = digitGroupSeparator
dGroup = digitalGroupSpacing
// negative or positive symbols (future-proof to take into account the future possibility
// to show a '+' sign for positive numbers)
pNeg = negativePositiveSignPlacement or signPlacement or signSymbolPlacement
// false default & true "+" shown on positive values.
showPositiveSign
```

**Comment by AlexandreBonneau** [comment]
This is fixed by PR #298
